examples/translation-chatbot/bot.py

```diff
@@ -44,18 +51,34 @@
 # We need to use a custom service here to yield LLM frames without saving
 # any context
 class TranslationProcessor(FrameProcessor):
-    def __init__(self, language):
+    """A processor that translates text frames from a source language to a target language."""
+
+    def __init__(self, in_language, out_language):
+        """Initialize the TranslationProcessor with source and target languages.
+
+        Args:
+            in_language (str): The language of the input text.
+            out_language (str): The language to translate the text into.
+        """
         super().__init__()
-        self._language = language
+        self._language = out_language
```

Comment:
Maybe name this as follows for consistency:
```suggestion
        self._out_language = out_language
```

And update the corresponding usage in this class.